### PR TITLE
KZOO-77: Merge E911 non-address fields

### PR DESCRIPTION
**Pull request by harenson**

### CODE REVIEW

File: kazoo_numbers/src/providers/knm_dash_e911.erl [hunk 1]
```diff
@@ -96,7 +96,8 @@ maybe_update_e911(PN) ->
             PN;
         'false' ->
             lager:debug("information has been changed: ~s", [kz_json:encode(E911)]),
-            NewE911 = maybe_update_e911(PN, E911),
+            %% kz_json:merge/1 invoked to update "non address" fields like `notification_contact_emails'
```
Comment (jamesaimonetti) [inline kazoo_numbers/src/providers/knm_dash_e911.erl]: It is arity-2 :p

**Comment by jamesaimonetti** [comment]
What about knm_telnyx_911?

**Comment by jamesaimonetti** [comment]
Also can we see a test call to confirm expected inputs + outputs?

File: kazoo_numbers/src/providers/knm_dash_e911.erl (hunk 1 above)
Comment (harenson) [reply kazoo_numbers/src/providers/knm_dash_e911.erl]: :D

**Comment by harenson** [comment]
knm_telnyx_911 uses a different approach, it doesn't merge old and new e911 objects, it just uses the new e911 object as the resulting object.

**Pull request merged**